### ISSUE TITLE
what sex is this saint?  

### ISSUE THREAD

**Issue by jsaintlauren** [srophe/srophe-app-data#477]
@nathangibson @davidamichelson @dlschwartz 

Any thoughts about this saint's sex?
http://wwwb.library.vanderbilt.edu/exist/apps/srophe/work/1488 
I am intrigued and puzzled.

**Comment by nathangibson** [comment]
To me she looks feminine from the use of ܗܘܬ, but am I missing something?

**Comment by dlschwartz** [comment]
I would go with Female

On Tue, Aug 9, 2016 at 3:11 PM, nathangibson <notifications@github.com>
wrote:

> To me she looks feminine from the use of ܗܘܬ, but am I missing something?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_srophe_srophe-2Dapp-2Ddata_issues_477-23issuecomment-2D238675778&d=CwMFaQ&c=ODFT-G5SujMiGrKuoJJjVg&r=QzRmQWHC9YV0c5AgrobproqvWFdcgVpw66BsP8UwAQ0&m=9-Gi9-CImgrgTyghcpGGXwr_5mVEl08oQvBiaFfL1v0&s=Q7QJOgILdI0SYHdlLxJJwE8xk3B4L3EcNb9Dk64oWBE&e=>,
> or mute the thread
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AEwNzstYzkSWTXrS3vUvfTDv82rzHaV-2Dks5qeN8HgaJpZM4JWWPz&d=CwMFaQ&c=ODFT-G5SujMiGrKuoJJjVg&r=QzRmQWHC9YV0c5AgrobproqvWFdcgVpw66BsP8UwAQ0&m=9-Gi9-CImgrgTyghcpGGXwr_5mVEl08oQvBiaFfL1v0&s=gm3Ku_tYvPYtG_gwIRXhsHA3U5BN1HoRyRiA8YSIuZs&e=>
> .
>



-- 
Dr. Daniel L. Schwartz
Associate Professor of History
Associate Director of the Initiative for Digital Humanities, Media, and
Culture <http://idhmc.tamu.edu/>

Texas A&M University
4236 TAMU
College Station, Texas 77843-4236

*Paideia and Cult <http://chs.harvard.edu/CHS/article/display/5813>:
Christian Initiation in Theodore of Mopsuestia* (Center for Hellenic
Studies, 2013)
www.syriaca.org <http://syriaca.org/>

**Comment by dlschwartz** [comment]
Can we close this issue?

**Comment by wlpotter** [comment]
I don't believe https://syriaca.org/person/1587 (who is commemorated in https://syriaca.org/work/1488) has been marked as female. @jsaintlauren is this the right sex for this saint?
